docs/pip_transition.md

```diff
@@ -0,0 +1,76 @@
+## Transition from using pip packages
+
+This documents the proposed transition to no longer using pip packages (fully debianised system)
+
+### Rationale
```

Comment:
I guess another one here is that some packages are installed via both deb and pip which causes `dpkg --verify` to freak out. You can see this here https://github.com/KanoComputing/kano-settings/pull/496/files

Reply:
Yeah. To note our verbal discussion: That was because `pip uninstall` will happily remove any pip packages, not just ones it installed itself. This is why I propose manually moving `/usr/local/lib/python2.7/dist-packages`: I couldn't convince myself that we could use `pip uninstall` reliably to uninstall the existing pip packages without breaking stuff.

Reply:
In fact, even though not confirmed, my suspicion was that `pip uninstall` was one of the reasons of the crash; when I stopped the installer between the uninstall phase it didn't properly handled the `dist-packages` locations. That shows it doesn't work as expected especially concerning handling existing `pip` depedencies.